src/snowcli/cli/project/util.py

```diff
@@ -2,6 +2,12 @@
 import os
 from typing import Optional
 
+IDENTIFIER = r'((?:"[^"]*(?:""[^"]*)*")|(?:[A-Za-z_][\w$]{0,254}))'
```

Comment:
This was _slightly_ modified when moved from `schema.py`. Had to make sure all the interior groups were non-capturing, and that the over-arching group _was_ capturing (i.e. no `?:`)